Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Pipfile

```diff
@@ -13,6 +13,8 @@ pytest-asyncio = "*"
 [packages]
 pytest = "*"
 requests = "*"
+cryptography = "*"
```

Comment:
was this added to make the unittests pass?

Reply:
removed